operators/operator_sdk/osdk-working-bundle-images.adoc

```diff
@@ -10,6 +10,14 @@ You can use the Operator SDK to package, deploy, and upgrade Operators in the bu
 
 include::modules/osdk-bundle-operator.adoc[leveloffset=+1]
 include::modules/osdk-deploy-olm.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../operators/understanding/olm-packaging-format.adoc#olm-file-based-catalogs_olm-packaging-format[File-based catalogs] in Operator Framework packaging format
+* xref:../../operators/admin/olm-managing-custom-catalogs.adoc/#olm-managing-custom-catalogs-fb[File-based catalogs] in Managing custom catalogs
```

Comment:
The link "[File-based catalogs](http://file.rdu.redhat.com/mipeter/osdk_4.11--osdocs-3531-fbc-run-bundle/operators/admin/olm-managing-custom-catalogs.adoc/.html#olm-managing-custom-catalogs-fb)"  return a 404 console

Reply:
Addressed. Thank you!